lib/model/ProfielModel.class.php

```diff
@@ -50,11 +50,11 @@ public static function existsUid($uid) {
 	}
 
 	public static function existsNick($nick) {
-		return Database::sqlExists(static::getTableName(), 'nickname = ?', array($nick));

```

Comment:
Dit is een van de weinige static convenience functies die nog over is. Je zou ze allemaal non-static kunnen maken.